### PR TITLE
Add google compute

### PR DESCRIPTION
**Pull request by victorursan**

### CODE REVIEW

File: cs-google/src/main/scala/io/cloudslang/content/google/services/compute/disks/DiskController.scala [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      .setSource(source)
+      .setInterface(interface)
+
+    deviceNameOpt match {
+      case Some(deviceName) => attachedDisk.setDeviceName(deviceName)
+      case _ => attachedDisk
+    }
+  }
+
+  def createAttachedDisk(boot: Boolean,
```
Comment (victorursan) [inline cs-google/src/main/scala/io/cloudslang/content/google/services/compute/disks/DiskController.scala]: method overloading ... you could have used a method with default values and pattern matching ( like the other methods)

**Comment by robert-sandor** [comment]
@victorursan can you take a look ?

File: cs-google/src/main/scala/io/cloudslang/content/google/services/compute/disks/DiskController.scala [hunk 2]
```diff
@@ -29,43 +29,37 @@ object DiskController {
...
       .setAutoDelete(autoDelete)
-      .setMode(mode)
-      .setSource(source)
-      .setInterface(interface)
+      .setMode(mountMode)
 
-    deviceNameOpt match {
-      case Some(deviceName) => attachedDisk.setDeviceName(deviceName)
-      case _ => attachedDisk
+    sourceOpt match {
```
Comment (victorursan) [inline cs-google/src/main/scala/io/cloudslang/content/google/services/compute/disks/DiskController.scala]: if you don't want to group all the optionals (keep one/ match) together, it's more elegant to use:  `sourceOpt.foreach(source => attachedDisk.setSource(source))`  ... in general when you do pattern matching you should be exhaustive

**Comment by victorursan** [comment]
:+1:

**Pull request merged**